estuary/utils/general.py

```diff
@@ -399,3 +399,37 @@ def set_story_labels(requested_node_label, results, reverse=False):
         idx += delta
         node_idx += delta
     return results
+
+
+def create_header(item_type, item_id, story_node_story_flow, backward):
```

Comment:
Could you please rename this to something more descriptive like `get_siblings_title`? `create_header` sounds too much like you're creating HTTP headers for an API query.

Reply:
`item_type` and `item_id` are non-descriptive. Also, why not just pass the `story_node` object directly instead of passing in two of it's parameters?

Reply:
Optional: I know it's less efficient but I'd prefer this function actually get the story_node_story_flow value instead of having it passed in. It'd make the function call simpler and cleaner and the function to get that value is quite light-weight.

Reply:
`backward` should default to `False` for convenience